src/MonitorProfiler/MainProfiler/MainProfiler.cpp

```diff
@@ -290,6 +291,11 @@ HRESULT MainProfiler::ProcessCallstackMessage()
         IfFailLogRet(eventProvider->WriteCallstack(stackState->GetStack()));
     }
 
+    //HACK See https://github.com/dotnet/runtime/issues/76704
+    // We sleep here for 200ms to ensure that our event is timestamped. Since we are on a dedicated message
+    // thread we should not be interfering with the app itself.
+    ThreadUtilities::Sleep(200);
```

Comment:
I'm assuming this value (`200`) should be the same as https://github.com/dotnet/dotnet-monitor/pull/2791/files#diff-f2ab9c8a7408cb463a090d005d24483a25ede050ee8f4ffe43693945d01e7e3cR99 (`250`)

Reply:
150 is the absolute minimum, since that's the actual timer in the framework. However, these are slightly different scenarios. For stacks, this will stall the scenario by this much time. For log testing, the idea is to speed up tests by creating a fake log value. I will consider changing this value if our stacks tests are very slow.